### ISSUE TITLE
Feature Request: Add Command Line Argument for setting Custom SkyrimSE/SKSE Path for client

### ISSUE THREAD

**Issue by 4cecoder** [tiltedphoques/TiltedEvolution#627]
## Description

TL;DR: Request to add command line arguments in Skyrim Together for specifying the path of the SkyrimSE.exe/SKSE. This enhancement could resolve issues experienced when launching SkyrimSE and SKSE on Linux through Wine, where the GUI selector currently underperforms.

**Comment by Dragonisser** [comment]
SKSE is Skyrim Script Extender, not Skyrim Special Edition.

**Comment by 4cecoder** [comment]
> SKSE is Skyrim Script Extender, not Skyrim Special Edition.

Yes that's true, yet I still would need a way to set the file path by passing a command line argument.

**Comment by 4cecoder** [comment]
Any possibility to add this feature?
